### PR TITLE
ports "Radial card hands #50729" from TG

### PR DESCRIPTION
**Pull request by Jumps0**
### Intent of your Pull Request

All Credit to **RaveRadbury** for coding, all I did was port it.

When viewing your hand of cards instead of being given a list menu you are now given a radial menu.
hFxSOkZYB2

#### Changelog

:cl:  RaveRadbury
tweak: Radial menu for hands of cards
/:cl:

### CODE REVIEW

**Comment by Jumps0** [comment]
i can't believe you've done this

**Comment by Jumps0** [comment]
ladies and gentlement, we got em

File: code/game/objects/items/toys.dm [hunk 1]
```diff
@@ -866,6 +848,18 @@
 	newobj.card_attack_verb = sourceobj.card_attack_verb
 	newobj.resistance_flags = sourceobj.resistance_flags
 
+/**
+  * This proc updates the sprite for when you create a hand of cards
+  */
+/obj/item/toy/cards/cardhand/proc/update_sprite()
```
Comment (Hopekz) [inline code/game/objects/items/toys.dm]:
```suggestion
/// This proc updates the sprite for when you create a hand of cards
/obj/item/toy/cards/cardhand/proc/update_sprite()
```

**Review by Hopekz — CHANGES_REQUESTED** [review]

**Comment by Jumps0** [comment]
HOW

**Pull request merged**